README.md

```diff
@@ -74,7 +74,7 @@ All fields are optional and ecs-logs will assume defaults if some are missing.
 
 The default source that ecs-logs uses is *stdin*, in most cases this is not what
 will be used in production but can be useful for development and testing
-purposes.  
+purposes.
```

Comment:
ah, whoops, will remove those.  Need to add markdown-mode as an exemption from clearing trailing whitespace